src/transformers/models/segformer/configuration_segformer.py

```diff
@@ -122,12 +126,14 @@ def __init__(
         layer_norm_eps=1e-6,
         decoder_hidden_size=256,
         is_encoder_decoder=False,
-        reshape_last_stage=True,
         semantic_loss_ignore_index=255,
         **kwargs
     ):
         super().__init__(**kwargs)
 
+        if "reshape_last_stage" in kwargs:
+            warnings.warn(DEPRECATION_WARNING, FutureWarning)
```

Comment:
This warning will be issued for the models in the Hub that contain this in their config. So, those configs should be updated to remove that attribute.

Reply:
Yes, can I remove it after we have a new version of Transformers?